assets/src/components/v2/train_crowding.tsx

```diff
@@ -0,0 +1,163 @@
+
+import CarNotCrowdedLeft from "../../../static/images/svgr_bundled/train_crowding/Car-NotCrowded-Left.svg";
+import CarSomeCrowdingLeft from "../../../static/images/svgr_bundled/train_crowding/Car-SomeCrowding-Left.svg";
+import CarCrowdedLeft from "../../../static/images/svgr_bundled/train_crowding/Car-Crowded-Left.svg";
+import CarNoDataLeft from "../../../static/images/svgr_bundled/train_crowding/Car-NoData-Left.svg";
+import CarClosedLeft from "../../../static/images/svgr_bundled/train_crowding/Car-Closed-Left.svg";
+
+import CarNotCrowdedMiddle from "../../../static/images/svgr_bundled/train_crowding/Car-NotCrowded-Middle.svg";
+import CarSomeCrowdingMiddle from "../../../static/images/svgr_bundled/train_crowding/Car-SomeCrowding-Middle.svg";
+import CarCrowdedMiddle from "../../../static/images/svgr_bundled/train_crowding/Car-Crowded-Middle.svg";
+import CarNoDataMiddle from "../../../static/images/svgr_bundled/train_crowding/Car-NoData-Middle.svg";
+import CarClosedMiddle from "../../../static/images/svgr_bundled/train_crowding/Car-Closed-Middle.svg";
+
+import CarNotCrowdedRight from "../../../static/images/svgr_bundled/train_crowding/Car-NotCrowded-Right.svg";
+import CarSomeCrowdingRight from "../../../static/images/svgr_bundled/train_crowding/Car-SomeCrowding-Right.svg";
+import CarCrowdedRight from "../../../static/images/svgr_bundled/train_crowding/Car-Crowded-Right.svg";
+import CarNoDataRight from "../../../static/images/svgr_bundled/train_crowding/Car-NoData-Right.svg";
+import CarClosedRight from "../../../static/images/svgr_bundled/train_crowding/Car-Closed-Right.svg";
+
+interface Props {
+  arrivalTime: string,
+  crowding: string[],
+  destination: string,
+  front_car_direction: string,
+  now: string,
+  platform_position: number
+}
+
+const lookupCarComponent = (occupancyStatus: string, frontCarDirection: string | boolean) => {
+  if (frontCarDirection) {
+    if (occupancyStatus == "not_crowded") return frontCarDirection == "left" ? CarNotCrowdedLeft : CarNotCrowdedRight
+    else if (occupancyStatus == "some_crowding") return frontCarDirection == "left" ? CarSomeCrowdingLeft : CarSomeCrowdingRight
+    else if (occupancyStatus == "crowded") return frontCarDirection == "left" ? CarCrowdedLeft : CarCrowdedRight
+    else if (occupancyStatus == "closed") return frontCarDirection == "left" ? CarClosedLeft : CarClosedRight
+    else return frontCarDirection == "left" ? CarNoDataLeft : CarNoDataRight
+  } else {
+    if (occupancyStatus == "not_crowded") return CarNotCrowdedMiddle
+    else if (occupancyStatus == "some_crowding") return CarSomeCrowdingMiddle
+    else if (occupancyStatus == "crowded") return CarCrowdedMiddle
+    else if (occupancyStatus == "closed") return CarClosedMiddle
+    else return CarNoDataMiddle
+  }
+}
+
+const TrainCrowding: React.ComponentType<Props> = ({
+      crowding,
+      destination,
+      platform_position,
+      front_car_direction,
```

Comment:
Consider renaming these to camel case for consistency (not on the backend but here where you grab them and assign to variables)
```suggestion
      platform_position: platformPosition,
      front_car_direction: frontCarDirection,
```

Reply:
Aha! I like it 👍